src/dymaptic.GeoBlazor.Core/GeoBlazorSettings.cs

```diff
@@ -14,15 +14,4 @@ public class GeoBlazorSettings
     ///     The GeoBlazor Pro License Key, generated on the licensing server.
     /// </summary>
     public string? LicenseKey { get; set; }
-    
-    /// <summary>
-    ///     The Application Name for a MAUI application. Not required for Blazor Server or Blazor WebAssembly.
-    /// </summary>
-    public string? MauiAppName { get; set; }
```

Comment:
seems unrelated, did you mean to remove these?

Reply:
looks like I got a new merge from `develop` mixed in here that wasn't already merged to v3, but yes, this is going away.